src/effects/VST/VSTEffect.cpp

```diff
@@ -1332,6 +1342,10 @@ bool VSTEffect::ValidateUI(EffectSettings &settings)
 
 bool VSTEffect::CloseUI()
 {
+   // TODO: now that the UI is owned/managed by the validator,
```

Comment:
ValidateUI, CloseUI, OnTimer, NeedIdle, NeedEditIdle, RefreshParameters, OnSizeWindow, OnSlider all move to Validator

TransferDataToWindow becomes UpdateUI of the Validator

Reply:
ok, done